config/locales/en.yml

```diff
@@ -1,8 +1,23 @@
 en:
   books:
+    new:
+      header: "Add a new book"
     index:
       add_new_book_link: "Add a new book"
       header: "Library Catalog"
+    edit:
+      header: "Edit this book"
+      link: "Edit"
+    create:
+      button: "Create Book"
+      flash_success: "Book added"
+    update:
+      button: "Update Book"
+      flash_success: "Book edited"
+    delete:
+      confirmation_text: "Are you sure?"
+      flash_success: "Book deleted"
+      link: "Delete"
```

Comment:
In a separate PR you can remove all the quotes. YAML tends to do quite well without them. Sorry I didn't catch this earlier!